docs/index.md

```diff
@@ -0,0 +1,8 @@
+# Pulp 3 CLI Docs
+
+The documentation for pulp-cli and pulp-glue has moved to a new place.
+
+Learn more about the new documentation project [here](https://discourse.pulpproject.org/t/unified-documentation-part-2/1188/2) or go to the new site:
+
+- [pulp-cli section](https://staging-docs.pulpproject.org/pulp-cli/docs/user/)
+- [pulp-glue section](https://staging-docs.pulpproject.org/pulp-glue/docs/dev/learn/architecture/)
```

Comment:
Looks good in general.
I'd point the users to the new location first and then provide a link to the why however.

Reply:
You mean, move the user directly to staging-docs (like [here](https://staging-docs.pulpproject.org/pulp-cli/docs/user/)) and edit the page there to contain the migration note?

Reply:
No, i just meant to shuffle the lines in this file around.

Whoever got here wants to know where it's now. Only a tiny portion of visitors may be interested in the why. So move the important links to the top.